scim2-sdk-client/src/main/java/com/unboundid/scim2/client/requests/RequestBuilder.java

```diff
@@ -275,4 +278,40 @@ protected Invocation.Builder buildRequest()
     }
     return builder;
   }
+
+  /**
+   * This method is equivalent to calling {@code asGenericScimResource()}. This
+   * is used by the request builders as a halfway step toward converting an
+   * object into a JSON payload. This minimizes the likelihood for improper JSON
+   * conversions, such as {@code null} fields being included in the JSON.
+   * <br><br>
+   *
+   * Before sending a request to a SCIM service, the Java object representing a
+   * SCIM resource must be converted into a string JSON payload. It's possible
+   * to simply pass in the Java object and rely on other libraries to serialize
+   * it into JSON later, but this approach requires projects to perform manual
+   * tuning of Jackson settings in the environment so that other libraries will
+   * serialize the data correctly. This can be burdensome to configure, and can
+   * result in problems with the HTTP request that are difficult to resolve.
+   * <br><br>
+   *
+   * Since any object may be used as the JSON payload, it is valuable to convert
+   * these SCIM resources into a JSON-like form before sending an HTTP request.
+   * GenericScimResource is well-suited for this, as it is a wrapper around a
+   * JSON ObjectNode that is more likely to result in a proper JSON string.
+   * <br><br>
+   *
+   * The behavior of the SCIM SDK's object mapper can be customized by updating
+   * fields in the {@link com.unboundid.scim2.common.utils.MapperFactory} class.
+   *
+   * @param resource  The SCIM resource that will be serialized into JSON.
+   * @return  An equivalent GenericScimResource.
+   *
+   * @since 4.0.1
+   */
+  @NotNull
+  protected GenericScimResource generify(@NotNull final ScimResource resource)
```

Comment:
I made a wrapper method for this so that there was a centralized spot that documents why a GenericScimResource is used.